USTests.py

```diff
@@ -272,6 +272,66 @@ def test_marriage_after_fourteen(self):
+        self.individual9 = Individual(id="I9", name="Jack Smith", gender="M", birthday="01 JAN 1990", death="NA")
+        self.individual10 = Individual(id="I10", name="Jill Smith", gender="F", birthday="5 DEC 1989", death="NA")
+
+    def test_death_before_birth(self):
+        self.assertTrue(US24([self.family1, self.family2], [self.individual1, self.individual2, self.individual3, self.individual4], False))
+        self.assertFalse(US24([self.family1, self.family2, self.family3], [self.individual1, self.individual2, self.individual3, self.individual4, self.individual5, self.individual6], False))
+        self.assertFalse(US24([self.family1, self.family3, self.family4], [self.individual1, self.individual2, self.individual3, self.individual4], False))
+        self.assertTrue(US24([self.family2, self.family4], [self.individual3, self.individual4, self.individual7, self.individual8], False))
+        self.assertTrue(US24([self.family2, self.family4, self.family5], [self.individual3, self.individual4, self.individual7, self.individual8, self.individual9, self.individual10], False))
+
+class TestUS25(unittest.TestCase):
+    """
+    Author: Zac Schuh
+    User Story: US25
+    Sprint: Sprint 3
+    """
+
+    def setUp(self):
+        self.family2 = Family("F1", "15 JUN 1995", "NA", "I1", "John Smith", "I2", "Jane Smith")  # Never had a child
+        self.family1 = Family("F2", "15 JUN 1995", "NA", "I1", "John Smith", "I2", "Jane Smith", ["I3"])  # One child only
+        self.family3 = Family("F3", "15 JUN 1995", "NA", "I1", "John Smith", "I2", "Jane Smith", ["I3", "I4"])  # Two different children
+        self.family4 = Family("F4", "15 JUN 1995", "NA", "I1", "John Smith", "I2", "Jane Smith", ["I3", "I4", "I5", "I6"])  # Two children share a name and birthday
+        self.family5 = Family("F5", "15 JUN 1995", "NA", "I1", "John Smith", "I2", "Jane Smith", ["I3", "I4", "I7", "I8"])  # Two children share a name but not a birthday
+        self.individual1 = Individual(id="I1", name="John Smith", gender="M", birthday="01 JAN 1960", death="NA")
+        self.individual2 = Individual(id="I2", name="Jane Smith", gender="F", birthday="01 JAN 1970", death="NA")
+        self.individual3 = Individual(id="I3", name="Jack Smith", gender="M", birthday="01 JAN 1996", death="NA")
+        self.individual4 = Individual(id="I4", name="Jill Smith", gender="F", birthday="03 MAR 2003", death="NA")
+        self.individual5 = Individual(id="I5", name="James Smith", gender="M", birthday="10 JUN 2001", death="NA")
+        self.individual6 = Individual(id="I6", name="James Smith", gender="M", birthday="10 JUN 2001", death="NA")
+        self.individual7 = Individual(id="I7", name="Elizabeth Smith", gender="F", birthday="5 DEC 2007", death="NA")
+        self.individual8 = Individual(id="I8", name="Elizabeth Smith", gender="F", birthday="5 DEC 2009", death="NA")
+
+    def test_death_before_birth(self):
```

Comment:
@zschuh Can you rename the test function to something related to what you are testing?